### DISCUSSION TITLE
PHP Fatal error:  Allowed memory size of 1073741824 bytes exhausted

### DISCUSSION THREAD

**Discussion by LTaooo** [livewire/livewire#7617] [7. Bugs (archived)]
### Livewire version

v3.2.6

### Laravel version

v10.34,1

### Which PHP version are you using?

PHP 8.2

### Repository or Wirebox URL

abc

### Additional information

1. Use octane with swoole5
2. Deploy in Docker

![image](https://github.com/livewire/livewire/assets/28125259/8926b5b3-e5ec-4326-a823-34bd94205c39)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ghost** [comment]
If you pass a `non-default data` type to the blade,
For example, some `nested` types can cause this problem.
Because `livewire` doesn't recognize it
You can try converting it into an `array`, I just encountered this problem
Please use the default type or customize it
😄
